src/business_logic/state/cached_state.rs

```diff
@@ -20,10 +20,10 @@ pub(crate) const UNINITIALIZED_CLASS_HASH: &[u8; 32] = b"\x00\x00\x00\x00\x00\x0
 
 #[derive(Debug, Clone, Default, Eq, Getters, MutGetters, PartialEq)]
 pub struct CachedState<T: StateReader + Clone> {
-    pub(crate) state_reader: T,
+    pub state_reader: T,
     #[getset(get = "pub", get_mut = "pub")]
-    pub(crate) cache: StateCache,
-    pub(crate) contract_classes: Option<ContractClassCache>,
+    pub cache: StateCache,
+    pub contract_classes: Option<ContractClassCache>,
```

Comment:
If you make this public, then there's no point in making getters...
I'd leave the getters for now.